test/Libraries/RevitTestServices/RevitNodeTestBase.cs

```diff
@@ -43,8 +43,7 @@ public override void Setup()
 
         protected override TestSessionConfiguration GetTestSessionConfiguration()
         {
-            var asmDir = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
-            return new TestSessionConfiguration(Path.GetFullPath(asmDir + @"\..\"), asmDir);
+            return new TestSessionConfiguration(Dynamo.Applications.DynamoRevitApp.DynamoCorePath);
```

Comment:
DynamoRevitDS has a mechanism to resolve the DynamoCore path, whereas version selector doesn't have this logic and hence it won't be able to find SystemTestServices assembly. When version selector is loaded it doesn't load DynamoRevitDS by default unless there is only one version of Dynamo Revit present on the system.